asapdiscovery-docking/asapdiscovery/docking/scripts/run_docking_oe.py

```diff
@@ -552,7 +635,23 @@ def main():
         ]
         mp_args.extend(new_args)
 
+    if args.verbose:
+        if len(skipped) > 0:
+            logger.warning(f"Skipped {len(skipped)} receptor/ligand pairs")
+            for s in skipped:
+                logger.warning("Skipped pair: " + s)
+
+    if len(mp_args) == 0:
+        raise ValueError(
+            "No MP args built, likely due to no xtals found, check logs and increase verbosity with --verbose for more info. "
+        )
+
+    logger.info(
+        f"MP args built, {len(mp_args)} total with {failures} failures, most likely due to skipped xtals"
+    )
```

Comment:
this is confusing if there are no failures
```suggestion
    if failures > 0:
          logger.info(
              f"MP args built, {len(mp_args)} total with {failures} failures, most likely due to skipped xtals.\n"
              "Use --verbose flag to find out more"
          )
    else:
          logger.info(
              f"{len(mp_args)} multiprocessing args built.
          )
```